pkg/apps/store/store.go

```diff
@@ -223,31 +258,37 @@ func generatePVCs(v types.PublicVersion, config *protoform.Config, kubeclient *k
 	return pvcs, nil
 }
 
-func generateImageTag(currentImage string, blackduck *v1.Blackduck) string {
-	if len(blackduck.Spec.ImageRegistries) > 0 {
-		imageName, err := util.GetImageName(currentImage)
+func generateImageTag(defaultImage string, imageRegistries []string, registryConfig *api.RegistryConfiguration) string {
+	if len(imageRegistries) > 0 {
+		imageName, err := util.GetImageName(defaultImage)
 		if err != nil {
-			return currentImage
+			return defaultImage
 		}
-		return getFullContainerNameFromImageRegistryConf(imageName, blackduck.Spec.ImageRegistries)
+		defaultImage = getFullContainerNameFromImageRegistryConf(imageName, imageRegistries, defaultImage)
 	}
 
-	if len(blackduck.Spec.RegistryConfiguration.Registry) > 0 && len(blackduck.Spec.RegistryConfiguration.Namespace) > 0 {
-		imageName, err := util.GetImageName(currentImage)
+	if len(registryConfig.Registry) > 0 && len(registryConfig.Namespace) > 0 {
+		getRegistryConfiguration(defaultImage, registryConfig)
```

Comment:
This should be "return get..." right?  Something should be down with the value from getRegistryConfiguration

Reply:
good catch.. fixed